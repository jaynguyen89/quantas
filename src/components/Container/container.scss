@use 'src/commons/variables' as vars;

.container {
    margin: auto;
    padding: 0;
    min-height: 100%;
    
    @media screen and (min-width: vars.$small-width) {
        width: 97%;
    }
    @media screen and (min-width: vars.$medium-width) {
        width: 90%;
    }
    @media screen and (min-width: vars.$desktop-width) {
        width: 80%;
    }
    @media screen and (min-width: vars.$large-width) {
        width: 80%;
    }
    @media screen and (min-width: vars.$wide-width) {
        width: 70%;
    }
    @media screen and (min-width: vars.$ultrawide-width) {
        width: 60%;
    }
}