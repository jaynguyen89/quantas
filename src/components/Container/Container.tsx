import React, { PropsWithChildren } from 'react';
import 'src/components/Container/container.scss';

const Container = ({ children }: PropsWithChildren) => {
    return (
        <div className='container'>
            {children}
        </div>
    );
};

export default Container;