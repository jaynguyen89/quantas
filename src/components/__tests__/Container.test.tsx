import React from 'react';
import { configure, shallow } from 'enzyme';
import Adapter from '@cfaester/enzyme-adapter-react-18';
import toJson from 'enzyme-to-json';
import Container from 'src/components/Container/Container';

configure({ adapter: new Adapter() });

describe('Loading.tsx', () => {
    it('renders correctly', () => {
        const wrapper = shallow(<Container />);
        expect(toJson(wrapper)).toMatchSnapshot();
    });
});