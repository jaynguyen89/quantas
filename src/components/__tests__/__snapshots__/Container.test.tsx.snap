// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading.tsx renders correctly 1`] = `
<div
  className="container"
/>
`;
